NuGet.config

```diff
@@ -10,5 +10,23 @@
 		<!-- Opt-In Automatic Package Restore in Visual Studio -->
 		<add key="automatic" value="True" />
 	</packageRestore>
+	
+	<packageSources>
+		<add key="Nuget" value="https://api.nuget.org/v3/index.json" />
+		<add key="Avalonia" value="https://nuget.avaloniaui.net/repository/outsystems-feed/" />
+		<add key="AvaloniaNightlyBuilds" value="https://www.myget.org/F/avalonia-ci/api/v2" />
```

Comment:
Is this public?

Reply:
If not, builds will fail

Reply:
yes, I've copy-pasted it from the webview repo